applications/vanilla/views/modules/helper_functions.php

```diff
@@ -87,7 +87,7 @@ function writePromotedContent($Content, $Sender) {
                 class="Title"><?php echo anchor(Gdn_Format::text(sliceString($Content['Name'], $Sender->TitleLimit), false), $ContentURL, 'DiscussionLink'); ?></div>
             <div class="Body">
                 <?php
-                echo anchor(strip_tags(Gdn_Format::to(sliceString($Content['Body'], $Sender->BodyLimit), $Content['Format'])), $ContentURL, 'BodyLink');
+                echo anchor(sliceString(Gdn_Format::PlainText($Content['Body']), $Sender->BodyLimit), $ContentURL, 'BodyLink');
```

Comment:
We just had another XSS merge where this actually needs an htmlspecialchars() around that sliceString().